server/app.js

```diff
@@ -576,7 +576,11 @@ app.get("/api/eye-spy/:id", tourService.getTour);
 
 /** Endpoint for retrieving asset information from the NetX DAMS */
 app.get("/api/objects/:id/assets", async (request, response) => {
-  const result = await damsService.getAssetByObjectId(request.params.id);
+  // The object number needs to go from a schema of "01.02.09" to "01_02_09"
+  // to conform with the DAMS naming schema for rendition names
+  const objectNumber = request.params.id.replace(/\./g, "_");
+  const result = await damsService.getAssetByObjectNumber(objectNumber);
```

Comment:
Based on some folder structure updates in our `Collection Website API` folder in NetX - which holds all of our Collection-available assets, we access the assets using the Object Number instead of Object ID.